packages/components/psammead-most-read/src/Rank/index.jsx

```diff
@@ -1,7 +1,7 @@
 import React from 'react';
 import { shape, string, oneOf, number } from 'prop-types';
 import styled from 'styled-components';
-import { getFoolscap } from '@bbc/gel-foundations/typography';
+import { getFoolscap, getTrafalgar } from '@bbc/gel-foundations/typography';
```

Comment:
I have a concern about both typography sets being pulled into the component for every single version of MostRead.  However, we're already currently pulling in many different typographic values into our components in general, with MostRead being a specific example of this.